pydis_site/apps/content/resources/guides/python-guides/fix-ssl-certificate.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Fixing an SSL Certificate Error.
+description: A guide on fixing an SSL certificate error.
```

Comment:
What does "an SSL certificate error" mean? Shouldn't we specify which one, and why?